### ISSUE TITLE
Enabling `autocreate` in preferences causes `architect/create` to throw an error on Sandbox start with custom function paths

### ISSUE THREAD

**Issue by andybee** [architect/architect#1432]
**Describe the issue**
If you enable `autocreate` on a project that has a route with a custom defined path to the function source code and no `config.arc` file present, `architect/create` will error with:

```
TypeError: Cannot read properties of undefined (reading 'runtimeAlias')
    at <project>/node_modules/@architect/create/src/write-configs/index.js:27:32
    at Array.forEach (<anonymous>)
    at writeArcConfigs (<project>/node_modules/@architect/create/src/write-configs/index.js:22:38)
    at create (<project>/node_modules/@architect/create/src/index.js:94:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Steps to reproduce**
1. Create a new Architect project and configure two routes in `app.arc`, making the `src` of one the default path of the other.
2. Add `preferences.arc` file and under `@create` add `autocreate true`.
3. Start Sandbox.

**Expected behavior**
Auto create would behave as it does without custom paths.

**Comment by ryanblock** [comment]
I see the issue here, thank you! Please stand by for the fix.

**Comment by ryanblock** [comment]
Fixed in 10.12.7!
